less/mixins.less

```diff
@@ -1 +1,26 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+
+// Breakpoints
+@phone: ~'(max-width: 500px)';
+
+// Button
+.buttonmixin(@bcolor, @fonter, @tcolor) {
+    text-align: center;
+    padding: 1.0rem 0.5rem;
+    border-radius: 0.6rem;
+    background-color: @bcolor;
+    font-size: @fonter;
+    color: @tcolor;
+}
+
+.headerOne()  {
```

Comment:
while this works mixins are better used for more generic styling that you would have been forced to write over and over again. in this case if it is applying to all of your H1's you could either address the selection in the reset or at the top of your less (remembering about imports) you could have 1 h1 selection and that would flow through the page

Reply:
Not the most useful...It would have been more useful if all my h1 's needed to be standardized.